frescobaldi_app/scorewiz/scoreproperties.py

```diff
@@ -290,6 +293,26 @@ def lySimpleMidiTempo(self, node):
         val = self.metronomeValue.currentText() or '60'
         return ly.dom.Tempo(dur, val, node)
 
+    # Blank staff
```

Comment:
nitpick: Since blank staff is included in the function names the comment is unneeded